src/routes/+page.svelte

```diff
@@ -8,21 +8,22 @@
 	</div>
 
 	<div id="details">
-		<h1 style="font-size: 35px; margin-top:3ch;">What is this for?</h1>
+		<h1 style="font-size: 35px; margin-top:4ch">How do I use Compakt?</h1>
 		<h1 style="font-size: 20px; margin-top:-1ch">
-			The Compakt app is a tool designed by the Multirotor Software Division <br />
-			to assist with drone testing days by providing data logging and retrieval <br />
-			capabilities for flight data, among other useful features.
+			There are two main pages to navigate between, the "Record" and the "Logs" pages. <br>
+			Record does exactly what the name suggests, that is where you will go to enter information <br>
+			and record your flight. Once a flight has been recorded, the information will be used to create a record in our database <br> 
+			where it can be viewed via the logs page. The created logs are sorted by most recent to oldest by default <br>
+			and can be expanded to show more detailed weather data. You can also manage any previously recorded <br>
+			flight log with options for exporting or deleting flight logs. <br>
 		</h1>
-		<h1 style="font-size: 35px; margin-top:5ch">How is it used?</h1>
+		<h1 style="font-size: 35px; margin-top:4ch">Key Information</h1>
 		<h1 style="font-size: 20px; margin-top:-1ch">
-			There are two main pages you will need to navigate between, the Flight Recorder and the Flight
-			Logs. <br />
-			The Flight Recorder does exactly what the name suggests, that is where you will go to enter information
-			and record your flight.<br />
-			Once a flight has been recorded, that information will be stored within the app, where it can be
-			viewed in the Flight Logs. <br />
-			From there, you can view, manage, and export any previously recorded flight logs. <br />
+			- Compakt is compatible with both mobile and desktop platforms that have a connection to the internet. <br>
+			- The weather API used to gather data is based on the general Rolla area, not specifically our testing location <br>
```

Comment:
Change this line to not mention Rolla, we may use it in other cities in the future. Make it more general, like:
```suggestion
			- The weather API used to gather data is based on the general area of the city provided, not specifically the testing location <br>
```